client/src/modules/dashboard/Events/components/EventForm.tsx

```diff
@@ -71,10 +71,10 @@ const EventForm: React.FC<IEventFormProps> = props => {
                     {venue.name}
                   </MenuItem>
                 ))}
+                <MenuItem>None</MenuItem>
```

Comment:
This is just to avoid breaking the create/update event forms until the UI work for this issue is done. 

I did see a warning in the browser for this: `A component is changing an uncontrolled input of type undefined to be controlled. Input elements should not switch from uncontrolled to controlled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component.`

What's the correct way to add a null/undefined value to a form like this? Empty string causes an error because gql is expecting an integer. I am new to React :)